From 612e58b45b84eda7e1f2a82fdfc006f1ae421cb5 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Tue, 2 Aug 2011 07:20:15 +0200
Subject: [PATCH] gperf: add missing.h that was present in the C version to
 the m4

This fixes:

src/load-fragment-gperf.c:413:51: error: 'RLIMIT_RTTIME' undeclared (first use in this function)
---
 src/load-fragment-gperf.gperf.m4 |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/load-fragment-gperf.gperf.m4 b/src/load-fragment-gperf.gperf.m4
index 650f444..8e52890 100644
--- a/src/load-fragment-gperf.gperf.m4
+++ b/src/load-fragment-gperf.gperf.m4
@@ -2,6 +2,7 @@
 #include <stddef.h>
 #include "conf-parser.h"
 #include "load-fragment.h"
+#include "missing.h"
 %}
 struct ConfigPerfItem;
 %null_strings
